home_work_12/calculator.js

```diff
@@ -0,0 +1,146 @@
+const calculator = document.querySelector("#calculator");
+const numbers = document.querySelectorAll(".item-number");
+const outputField = document.getElementById("result");
+
+let firstValue = "";
+let mathOperator = "";
+let secondValue = "";
+let equals = "";
+let dot = "";
+
+function fadeCalculatorButton(btnValue) {
+  const clickedNumber = document.getElementById(btnValue);
+  clickedNumber.classList.add("fade-number");
+
+  setTimeout(() => {
+    clickedNumber.classList.remove("fade-number");
+  }, 0);
+}
+
+function getResulOperations() {
+  let result;
+  if (mathOperator === "+") {
+    result = +firstValue + +secondValue;
+    return parseFloat(result.toFixed(10));
+  }
+  if (mathOperator === "-") {
+    result = +firstValue - +secondValue;
+    return parseFloat(result.toFixed(10));
+  }
+  if (mathOperator === "x" || mathOperator === "*") {
+    result = +firstValue * +secondValue;
+    return parseFloat(result.toFixed(10));
+  }
+  if (mathOperator === "÷" || mathOperator === "/") {
+    +secondValue === 0
+      ? (result = "Error: Invalid operation")
+      : (result = +firstValue / +secondValue);
+    return parseFloat(result.toFixed(10));
```

Comment:
You should think about it once again, if the result will be string, there will be an error here trying to convert string to fixed
<img width="624" alt="image" src="https://user-images.githubusercontent.com/84010501/198872479-c861610e-ad57-45af-9f35-b806d4a08776.png">